tests/run/fused_def.pyx

```diff
@@ -511,3 +513,12 @@ def stress_test_thread_safety(n_threads):
     for t in threads:
         t.join()
     assert not failed_list, len(failed_list)
+
+def test_signature(arg1, arg2):
```

Comment:
This is not a fused function.

Reply:
That's kind of obvious now you point it out....